### PR TITLE
Update Hebrew translation

### PR DESCRIPTION
**Pull request by dinuto**

### CODE REVIEW

**Comment by micwoj92** [comment]
@HypeGameMode

File: Langs/040D.lang [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 Version=2.3.1.1
+[Font]
+Extra Large=24
+Large=18
+Medium=10
+Small=8.5
+Name=Arial
 [Strings]
+Enable enhanced accessibility?=לאפשר נגישות משופרת?
+This new Enhanced Accessibility functionality is designed for the visually impaired, in which most of the program interface can be used by voice and keyboard shortcuts. Activate?=This new Enhanced Accessibility functionality is designed for the visually impaired, in which most of the program interface can be used by voice and keyboard shortcuts. Activate?
```
Comment (micwoj92) [inline Langs/040D.lang:14]: You didn't translate this one

Comment (dinuto) [reply Langs/040D.lang]: Thanks, fixed!

**Comment by HypeGameMode** [comment]
I don't think it's fixed hebrew but I don't have other ideas hoe to say that message different so it's OK from me.

**Review by rcmaehl — APPROVED** [review]

**Pull request merged**